pkg/config/testdata/containers_comment.conf

```diff
@@ -18,6 +18,7 @@
 # environment variables to conmon or the runtime.
 # env = [
 #     "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+#    "TERM=xterm",
```

Comment:
spacing/tab issue here?  Line 20 and 21 are off by a space in the GitHub review pane.